beaker_kernel/kernel.py

```diff
@@ -429,8 +429,9 @@ def log(self, event_type: str, content, parent_header=None):
             "event": event_type,
             "body": content,
         }
+        message_type = "debug_event" if debug else "log_event"
         message = self.server.make_multipart_message(
-            msg_type="debug_event",
+            msg_type=message_type,
```

Comment:
`debug_event` is [defined as part of the jupyter message protocol](https://jupyter-client.readthedocs.io/en/stable/messaging.html#debug-event) for these types of messages, which in turn is an implementation of the [DAP protocol](https://microsoft.github.io/debug-adapter-protocol/).

If you need to track a "log level", that should probably be part of the "body" inside message_content.

That said, I'm not sure if this is needed, because I don't think "debug" messages will ever be sent if `beakerSession.value?.activeContext?.info?.debug` is not already true. So we shouldn't need to filter on the UI side something that is already filtered/controled on the back end.
(Because `beakerSession.value?.activeContext?.info?.debug` === `self.debug_enabled` at all times, I think)